database/engine/README.md

```diff
@@ -63,6 +62,14 @@ All DB engine instances, for localhost and all other streaming recipient nodes i
 The above values are the default and minimum values for Page Cache size and DB engine disk space quota. Both numbers are
 in **MiB**. All DB engine instances will allocate the configured resources separately.
 
+[**See our database engine calculator**](https://learn.netdata.cloud/docs/agent/database/calculator) to help you
+correctly set `dbengine disk space` based on your needs. The calculator gives an accurate estimate based on how many
+slave nodes you have, how many metrics your Agent collects, and more.
+
```

Comment:
Is something missing here?

Reply:
I forgot to delete this paragraph. Thanks for catching it.